inc/class-scripts.php

```diff
@@ -382,4 +427,116 @@ public function add_body_class_container_boxed($classes) {
 
 		return $classes;
 	}
+
+	/**
+	 * Checks if the current screen is a WP Ultimo admin page.
+	 *
+	 * @since 2.0.0
+	 *
+	 * @param \WP_Screen $screen The current screen object.
+	 * @return bool
+	 */
+	private function is_wp_ultimo_admin_page($screen) {
+		// Check if the screen ID contains 'wp-ultimo' or if it's a WP Ultimo admin page
+		return (
+			strpos($screen->id, 'wp-ultimo') !== false ||
+			strpos($screen->id, 'wu-') !== false ||
+			(isset($_GET['page']) && strpos($_GET['page'], 'wu-') === 0)
+		);
+	}
+
+	/**
+	 * Checks if the current screen is a WP Ultimo list page that needs the list table scripts.
+	 *
+	 * @since 2.0.0
+	 *
+	 * @param \WP_Screen $screen The current screen object.
+	 * @return bool
+	 */
+	private function is_wp_ultimo_list_page($screen) {
+		// Pages that use list tables
+		$list_pages = array(
+			'wu-memberships',
+			'wu-products',
+			'wu-customers',
+			'wu-payments',
+			'wu-discount-codes',
+			'wu-domain-mappings',
+			'wu-sites',
+		);
+
+		return $this->is_wp_ultimo_admin_page($screen) &&
+			   isset($_GET['page']) &&
+			   in_array($_GET['page'], $list_pages);
```

Comment:
```suggestion
			   in_array($_GET['page'], $list_pages, true);
```
add true to third param to enable strict mode.